src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -203,25 +203,26 @@ function TextArea({
 
     const textarea = textareaRef.current!;
     const listenerCollection = new ListenerCollection();
+    const checkShouldOpenPopover = () => {
```

Comment:
I think this would be better with a name that clarifies what is being checked for, if we can find something not _too_ verbose. Maybe `checkForMentionAtCaret`?

Reply:
Yeah, I like that.